code/API_definitions/network-access-management.yaml

```diff
@@ -1,3273 +1,3456 @@
+
+    | Scope | Description | Inherits
+    |-------|-------------|---------
+    | **network-access-management:service-sites:read** | List a service owner's service sites
+    | **network-access-management:service-sites.location:read** | Read a service site's physical location | network-access-management:service-sites:read
+    | **network-access-management:devices:read** | List a service owner's devices
+    | **network-access-management:devices.hardware-address:read** | Read a service owner's device's hardware address | network-access-management:devices:read
+    | **network-access-management:devices:write** | Modify which networks are configured on a service owner's devices | network-access-management:devices:read
+    | **network-access-management:isolated-networks:read** | List a service owner's isolated networks created by the calling API client
+    | **network-access-management:isolated-networks:write** | Create or modify a service owner's isolated networks created by the calling API client | network-access-management:isolated-networks:read
+    | **network-access-management:isolated-networks.primary:read** | Access information about a service owner's primary isolated network
+    | **network-access-management:isolated-networks.primary:write** | Modify a service owner's primary isolated network | network-access-management:isolated-networks.primary:read
+    | **network-access-management:isolated-networks.auxiliary:read** | List a service owner's auxiliary isolated networks | network-access-management:isolated-networks:read
+    | **network-access-management:isolated-networks.auxiliary:write** | Create or modify a service owner's auxiliary isolated networks | network-access-management:isolated-networks.auxiliary:read<br />network-access-management:isolated-networks:write
+    | **network-access-management:reboot-requests:read** | List a service owner's reboot requests created by the calling API client
+    | **network-access-management:reboot-requests:write** | Create or modify a service owner's reboot request created by the calling API client | network-access-management:reboot-request:read
+
+    ## General Usage
+    This API is intended for creating, modifying, or deleting isolated networks using the `/isolated-networks` endpoints
+    to build access detail configurations. The `/service-sites` endpoints are used for iterating over a service owner's
+    locations. The exact location may be requested with an additional scope to assist with reconciling information about
+    service sites when the API client operator has an existing relationship with the service owner. Similarly, the
+    `/devices` endpoints are used for iterating over and configuring a service owner's devices.  The hardware address
+    may be requested for the same purpose of reconciling information from another source (e.g., database, persons with
+    physical device access, etc.).
+
+    ## Optional Modes
```

Comment:
This naked heading without any text and just one sub-heading should either have text or be combined with Default Device.

Reply:
Dropped this title and set default device to level 2.